app/components/Channels/CloseChannelDialog.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
```

Comment:
Could we call this `ChannelCloseDialog` rather than `CloseChannelDialog` since all of the other related Channel components are prefixed `Channel`

Reply:
done